samples/deploy/crd/v1beta1/ApisixRoute.yaml

```diff
@@ -24,21 +24,27 @@ spec:
     - JSONPath: .spec.http[].match.hosts
       name: Hosts
       type: string
+      priority: 0
     - JSONPath: .spec.http[].match.paths
       name: URIs
       type: string
-    - JSONPath: .spec.http[].match.backends[].serviceName
-      name: Target Service
+      priority: 0
+    - JSONPath: .spec.http[].backend.serviceName
```

Comment:
the `JSONPath` is a simple string,  I just show the `.spec.http[].backend.serviceName`